iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/model/HouseTable.java

```diff
@@ -10,6 +9,7 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
```

Comment:
lets avoid setter pattern, we want the object to be immutable

Reply:
Lets discuss in https://github.com/linkedin/openhouse/pull/187#discussion_r1750715536